docs/tutorials/getting_started.md

```diff
@@ -61,118 +65,229 @@ directory:
 `mkdir -p /home/ubuntu/.local/share`
 ```
 
-## 3. Deploy SD-Core
+## 3. Install Terraform
 
+From your terminal, install Terraform.
 
-Create a Juju model named `core`:
+```console
+sudo snap install terraform --classic
+```
+
+## 4. Create Terraform module
+
+On the host machine create a new folder called `terraform`:
```

Comment:
I think I would say `directory` instead of `folder` but I don't know what the right term to use in English is.
(Same in line 84, 169)

Reply:
I think they essentially mean the same. Purists might say that the folder is a graphical representation of a directory, but to me it really boils down to the Potahto-Potato problem :)
Changed to `directory` ;)